botlib abuses strcpy (source and dest overlap), and the strcpy function for 64
bit intel CPU's in the latest glibc, does not like this causing the bots to not
load.

See: http://bugzilla.redhat.com/show_bug.cgi?id=526338
See: https://bugzilla.icculus.org/show_bug.cgi?id=4331

Index: openarena-engine-0.8.1/code/botlib/l_precomp.c
===================================================================
--- openarena-engine-0.8.1.orig/code/botlib/l_precomp.c
+++ openarena-engine-0.8.1/code/botlib/l_precomp.c
@@ -948,7 +948,7 @@ void PC_ConvertPath(char *path)
 		if ((*ptr == '\\' || *ptr == '/') &&
 				(*(ptr+1) == '\\' || *(ptr+1) == '/'))
 		{
-			strcpy(ptr, ptr+1);
+			memmove(ptr, ptr+1, strlen(ptr));
 		} //end if
 		else
 		{
Index: openarena-engine-0.8.1/code/botlib/l_script.c
===================================================================
--- openarena-engine-0.8.1.orig/code/botlib/l_script.c
+++ openarena-engine-0.8.1/code/botlib/l_script.c
@@ -1118,7 +1118,7 @@ void StripDoubleQuotes(char *string)
 {
 	if (*string == '\"')
 	{
-		strcpy(string, string+1);
+		memmove(string, string+1, strlen(string));
 	} //end if
 	if (string[strlen(string)-1] == '\"')
 	{
@@ -1135,7 +1135,7 @@ void StripSingleQuotes(char *string)
 {
 	if (*string == '\'')
 	{
-		strcpy(string, string+1);
+		memmove(string, string+1, strlen(string));
 	} //end if
 	if (string[strlen(string)-1] == '\'')
 	{
